src/lib/components/banner/banner-temporal-version.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+  import { isRecommendedVersionNewer } from '$lib/utilities/version-check';
+  import Banner from './banner.svelte';
+
+  export let cluster: ClusterInformation;
+
+  const severities = {
+    High: 'high',
+    Medium: 'medium',
+    Low: 'low',
+  } as const;
+
+  $: recommended = cluster?.versionInfo?.recommended;
```

Comment:
Do these all need to be reactive? Ideally a new `cluster` is being passed in, right?

Reply:
yea no need anymore, updated. I think it used to come from the store.